osu.Game/IO/MigratableStorage.cs

```diff
@@ -101,15 +108,22 @@ protected void CopyRecursive(DirectoryInfo source, DirectoryInfo destination, bo
                 if (topLevelExcludes && IgnoreFiles.Contains(fi.Name))
                     continue;
 
-                AttemptOperation(() => fi.CopyTo(Path.Combine(destination.FullName, fi.Name), true));
+                if (!AttemptOperation(() => fi.CopyTo(Path.Combine(destination.FullName, fi.Name), false)))
```

Comment:
Is this change from `true` to `false` intentional, or was it done just for testing purposes?

Reply:
Without this it would cause the exception in the linked failure.

Reply:
How? The linked failure is a file not found exception. This is an overwrite parameter?

Reply:
This is... an unintended change. I got the brackets wrong, and thought this was setting the parameter on `AttemptOperation`.